tektoncd/pipeline
# feat: Artifacts phase1 poc

- This is a poc work for [TEP-0147](https://github.com/tektoncd/community/pull/1125) artifacts phase 1

 ## Test Plan 
 
  - `ko apply -R -f config` in the project workspace
  - run `ko create -f step-artifacts.yaml` in `examples/v1/taskruns` folder
  - run `tkn logs tr` and find the tr just created, and see the logs 


```
